detekt.yml

```diff
@@ -7,3 +7,8 @@ comments:
     active: false
   CommentOverPrivateProperty:
     active: false
+
+  style:
+    ForbiddenComment:
+      # Allow TODO and FIXME
+      values: [ 'STOPSHIP:' ]
```

Comment:
Style should be top-level

```suggestion
style:
  ForbiddenComment:
    # Allow TODO and FIXME
    values: [ 'STOPSHIP:' ]
```

Reply:
Whoops thought it was... :speak_no_evil: